x-pack/plugins/security_solution/server/endpoint/routes/metadata/handlers.ts

```diff
@@ -228,25 +210,21 @@ export async function getHostData(
     throw Boom.badRequest('esClient not found');
   }
 
-  const hostResult = await getHostMetaData(metadataRequestContext, id, queryStrategyVersion);
+  const hostMetadata = await getHostMetaData(metadataRequestContext, id);
 
-  if (!hostResult) {
+  if (!hostMetadata) {
     return undefined;
   }
 
-  const agent = await findAgent(metadataRequestContext, hostResult.metadata);
+  const agent = await findAgent(metadataRequestContext, hostMetadata);
 
   if (agent && !agent.active) {
     throw Boom.badRequest('the requested endpoint is unenrolled');
   }
 
-  const metadata = await enrichHostMetadata(
-    hostResult.metadata,
-    metadataRequestContext,
-    hostResult.query_strategy_version
-  );
+  const metadata = await enrichHostMetadata(hostMetadata, metadataRequestContext);
 
-  return { ...metadata, query_strategy_version: hostResult.query_strategy_version };
+  return { ...metadata };
```

Comment:
Should this now be `return metadata;` ? Is there a meaningful difference?

Reply:
good catch, should be just metadata.